lib/material_slider/material_slider.html

```diff
@@ -0,0 +1,30 @@
+<!--
+ Copyright (c) 2016, the Dart project authors.  Please see the AUTHORS file
+ for details. All rights reserved. Use of this source code is governed by a
+ BSD-style license that can be found in the LICENSE file.
+-->
```

Comment:
Do you really want the copyright notice as a comment inside the generated template? (I can't recall if this is what has been done elsewhere.)

Reply:
Ah, I thought that I'd seen some .html files w/o that copyright comment, but it seems that I'm mistaken.